Lib/unittest/test/testmock/testmagicmethods.py

```diff
@@ -280,6 +281,10 @@ def test_magicmock_defaults(self):
         self.assertEqual(hash(mock), object.__hash__(mock))
         self.assertEqual(str(mock), object.__str__(mock))
         self.assertTrue(bool(mock))
+        self.assertTrue(round(mock))
+        self.assertTrue(math.trunc(mock))
+        self.assertTrue(math.floor(mock))
+        self.assertTrue(math.ceil(mock))
```

Comment:
The result of round(), trunc(), etc. are a number, not a boolean. You should use assertEqual(..., value) instead.